### ISSUE TITLE
Use dashboard name / desc instead of generic text for meta title / description on dashboards

### ISSUE THREAD

**Issue by abalone23** [influxdata/ui#5254]
**Proposal:**
When adding dashboard links in Slack, it uses the generic meta title/description instead of the dashboard name/desc. It would be helpful to use the dashboard name/desc so people know what dashboard link it's referring to.

**Current behavior:**
Pasting a dashboard link in Slack looks like:
<img width="667" alt="Screen Shot 2022-08-01 at 11 32 36 AM" src="https://user-images.githubusercontent.com/330044/182219482-b3f98d07-8ba2-4fee-95ab-be2398414051.png">

**Desired behavior:**
It should display:
>[DASHBOARD_NAME]
[DASHBOARD DESCRIPTION]

**Alternatives considered:**
Make sure the person posting the link explains what the dashboard is before posting the link.

**Use case:**
Dashboards links pointing to many different dashboards across many different clusters are constantly being posted, sometimes making it hard to determine what they are referring to.

**Comment by ChitlangeSahas** [comment]
This is an awesome enhancement suggestion.

I did some digging and here are my findings.

**TLDR:** it might not be possible to implement this without implementing Server Side Rendering in our Architecture.

Reasoning: 

By nature the indexer that SEO's use only looks for the meta tags inside `index.html` file.

Any tags/metadata properties we change during runtime of the app doesn’t get added until later, so the indexer won't see them. And hence we see the **default title** (InfluxDB cloud) and description when pasting the links in slack or any [[metadata tag checker.](https://metatags.io/)](https://metatags.io/)

There seems to be a work around without needing a Server Side rendering (using [[react-snap](https://github.com/stereobooster/react-snap)](https://github.com/stereobooster/react-snap) and `react-helmet` to change tags on runtime), but I wasn’t able to make it work *plus* the libraries that make this possible are really out of date and not actively worked on. (seems like a bad idea to bring them into a production quality app like ours)

**Comment by kristinarobinson** [comment]
Not currently supported in architecture
